examples/components/bundled-rsbuild/wasmcloud.toml

```
@@ -0,0 +1,12 @@
+name = "typescript-bundled-rsbuild"
```

```suggestion
name = "bundled-rsbuild"
```

NIT: feel free to take this or leave it